### PR TITLE
Removed Linux as Visual Studio is not supported on Linux.

### PR DESCRIPTION
**Pull request by 18-F-cali**
[Internal Review](https://review.docs.microsoft.com/en-us/aspnet/core/blazor/?view=aspnetcore-6.0&branch=pr-en-us-25372)

### CODE REVIEW

**Review by wadepickett — COMMENTED** [review]
@guardrex, Maybe you meant for Linux to stay in here, but wanted to refer to the "Visual Studio Family" or "Visual Studio, Visual Studio for Mac and **Visual Studio Code**"?  "Visual Studio" is a particular product name rather than the family of products .  I see your link went to the family of products page.

**Comment by guardrex** [comment]
We should get @danroth27 to approve this change, as I think he wrote that ....... *and I don't like mess'in too much with what the boss gives me!* 😆 lol

**Comment by guardrex** [comment]
I think he was just saying "VS on Windows" ....... and also "on Linux" ... and also "on macOS" ... not VS on Win, Linux, and macOS. I think we could go with something like ...

> Stay productive on Windows, Linux, and macOS with a development environment, such as \[Visual Studio](https://visualstudio.microsoft.com) or \[Visual Studio Code](https://code.visualstudio.com/).

File: aspnetcore/blazor/index.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Using .NET for client-side web development offers the following advantages:
 * Leverage the existing .NET ecosystem of [.NET libraries](/dotnet/standard/class-libraries).
 * Share app logic across server and client.
 * Benefit from .NET's performance, reliability, and security.
-* Stay productive with [Visual Studio](https://visualstudio.microsoft.com) on Windows, Linux, and macOS.
+* Stay productive with [Visual Studio](https://visualstudio.microsoft.com) on Windows, and macOS.
```
Comment (guardrex) [inline aspnetcore/blazor/index.md:43]:
```suggestion
* Stay productive on Windows, Linux, or macOS with a development environment, such as [Visual Studio](https://visualstudio.microsoft.com/) or [Visual Studio Code](https://code.visualstudio.com/).
```

**Comment by guardrex** [comment]
Let's go ahead with this, and Dan may provide further guidance for a patch if needed.

Thanks @18-F-cali! :rocket:

**Review by danroth27 — APPROVED** [review]

**Pull request merged**